var/spack/repos/builtin/packages/libristra/package.py

```diff
@@ -0,0 +1,48 @@
+# Copyright 2013-2019 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+
+from spack import *
+
+
+class Libristra(CMakePackage):
+    """ The ristra library is a set of support utilities for
+    ristra codes, including simple mathematical operations, physical
+    units, and input parsing
+    """
+    homepage = 'https://github.com/laristra/libristra'
+    url = 'https://github.com/laristra/libristra/archive/master.zip'
+    git = 'https://github.com/laristra/libristra.git'
+    tags = ['ristra']
```

Comment:
Is there any other package that would be tagged the same? If not please remove the tag. Right now we are using those to group together set of packages, for instance we have a `build-tools` tag or `proxy-app` etc.

Reply:
There will be others but we can make later PRs then